src/client/components/ConsentCard.tsx

```diff
@@ -154,66 +161,88 @@ const frequencyStyles = css`
   padding: ${space[2]}px 0;
 `;
 
-const idColor = (id: string) => {
-  if (/today|morning/.test(id)) {
-    return news[400];
-  }
-  if (id === 'the-long-read') {
-    return brand[400];
-  }
-  if (id === 'green-light') {
-    return news[400];
-  }
-  if (id === 'bookmarks') {
-    return culture[400];
-  }
-  if (id === 'word-of-mouth' || id === 'the-guide-staying-in') {
-    return lifestyle[400];
-  }
-  return brand[400];
-};
+type HeadingLevel = 1 | 2 | 3 | 4 | 5;
 
-export const NewsletterCard: FunctionComponent<NewsletterCardProps> = (
-  props,
-) => {
-  const { description, frequency, name, nameId } = props.newsletter;
+interface ConsentCardProps {
+  title: string;
+  titleLevel?: HeadingLevel;
+  description: string;
+  id: string;
+  defaultChecked?: boolean;
+  imagePath?: string;
+  highlightColor?: string;
+  frequency?: string;
+  hiddenInput?: boolean;
+  cssOverrides?: SerializedStyles;
+}
+
+export const ConsentCard: FunctionComponent<ConsentCardProps> = ({
+  title,
+  titleLevel = 1,
+  description,
+  id,
+  defaultChecked,
+  imagePath,
+  highlightColor = brand[400],
+  frequency,
+  hiddenInput = false,
+  cssOverrides,
+}) => {
+  const TitleHeadingTag = `h${titleLevel}` as const;
+  const frequencyHeadingLevel = Math.min(titleLevel + 1, 5) as HeadingLevel;
+  const FrequencyHeadingTag = `h${frequencyHeadingLevel}` as const;
+  const checkboxChildren = [
+    <Checkbox
+      value={id}
+      cssOverrides={checkBoxBackgroundColorBugFix}
+      label="Yes, sign me up"
+      defaultChecked={defaultChecked}
+      key="visible"
+    />,
+  ];
+
+  if (hiddenInput) {
+    /**
+     * if the Checkbox is unchecked, this hidden empty value will be sent in
+     * form submit POST, to signal possible unsubscribe event
+     */
+    checkboxChildren.push(
+      <input type="hidden" name={id} value="" key="hidden" />,
+    );
+  }
```

Comment:
IIRC this was required to allow newsletters to unsubscribe, and the way html forms worked meant that if the checkbox was unchecked it wouldn't send the name/id in the POST request, so by using a hidden input, it would include the name/id in the post request even if the checkbox itself was unchecked.